pkgs/development/python-modules/youtube_search/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchFromGitHub, requests }:
+buildPythonPackage rec {
+  pname = "youtube_search";
+  version = "unstable-2021-02-27";
+
+  src = fetchFromGitHub {
+    owner = "joetats";
+    repo = pname;
+    rev = "886fe1b16c829215ee0984b6859f874b4a30d875";
+    sha256 = "sha256-3ECJ6iHNzx5PLgpTFraFzAYbKnyMYRf/iJ0zajU+hlo=";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  doCheck = false;
```

Comment:
Why did you disable tests? Please add a comment.

Also pythonImportsCheck missing.

Reply:
Hi, thank you for the reviews. I decided to disable checks as I get the following error when building without:

```python
FAILED (errors=1)
Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
error: Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
error: builder for '/nix/store/z0skj0gp3gn5ym0gwiak5c94y1vh4mm0-python3.8-youtube_search-unstable-2021-02-27.drv' failed with exit code 1;
       last 10 log lines:
       >     from .youtube_search import YoutubeSearch
       > ImportError: attempted relative import with no known parent package
       >
       >
       > ----------------------------------------------------------------------
       > Ran 1 test in 0.001s
       >
       > FAILED (errors=1)
       > Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
       > error: Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
       For full logs, run 'nix log /nix/store/z0skj0gp3gn5ym0gwiak5c94y1vh4mm0-python3.8-youtube_search-unstable-2021-02-27.drv'.
error: 1 dependencies of derivation '/nix/store/n63hkwgw5jm340zfsxrdb8vij9m3f82q-ytmdl-2021.05.26.drv' failed to build
```

Is there an option or patch I should be using to mitigate this error?